### PR TITLE
Fixes MongoDb Deprecation Warning for Version > 3.6.7

### LINKED ISSUE #135: Fix MongoDB DeprecationWarning: collection.findOneAndUpdate option [returnOriginal]
CONTEXT: RateLimiterMongo works without warning for Mongo Client v4+ with [this check](https://github.com/animir/node-rate-limiter-flexible/blob/master/lib/RateLimiterMongo.js#L172). But deprecation was introduced before v4, so some v3.x affected also.

TODO: Find exact version, when deprecation is added and improve mongoDB driver version check in [the condition](https://github.com/animir/node-rate-limiter-flexible/blob/master/lib/RateLimiterMongo.js#L172).

### PR DESCRIPTION
**Pull request by pavittarx**
Fixes #135 
- I have run the tests, and they pass successfully.

### CODE REVIEW

**Comment by pavittarx** [comment]
@animir I have made changes as suggested. Please have a look. :)

File: lib/RateLimiterMongo.js [hunk 1]
```diff
@@ -8,17 +8,18 @@ const RateLimiterRes = require('./RateLimiterRes');
...
-      ({ version } = client.topology.s.options.metadata.driver);
-    }
-    const majorVersion = parseInt(version);
+    const _client = client.client ? client.client : client;
 
-    return majorVersion;
+    const { version } = _client.topology.s.options.metadata.driver;
+    const _v = version.split('.').map(v => parseInt(v));
+
+    return {
```
[the remark this answers is not in the document]
Comment (pavittarx) [reply lib/RateLimiterMongo.js:16]: Done

**Comment by animir** [comment]
@pavittarx Nice! There is `package-lock.json` committed. We should remove it. Could you re-create this PR without `package-lock.json`? Also, you can pull changes from the master brunch, I've just added `package-lock.json` to `.gitignore`, so it can't be committed anymore.

**Comment by pavittarx** [comment]
@animir surely will do, been a little busy, 
Will it be okay, if I merge changes from main on the current branch & remove `package.json`?

**Comment by animir** [comment]
@pavittarx We should keep git history as small as possible, so removing is not a good option. Could you re-create this PR from scratch? It would require to create another branch from master, commit all changes you prepared and create a new PR.

**Pull request closed without merging**